itglue/client.go

```diff
@@ -0,0 +1,230 @@
+package usp_itglue
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net"
+	"net/http"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/refractionPOINT/go-uspclient"
+	"github.com/refractionPOINT/go-uspclient/protocol"
+	"github.com/refractionPOINT/usp-adapters/utils"
+)
+
+const (
+	logsURL = "/logs"
+)
+
+type opRequest struct {
+	Limit     int    `json:"page[size],omitempty"`
+	StartTime string `json:"filter[created_at],omitempty"`
+	Sort      string `json:"sort,omitempty"`
+}
+
+var URL = map[string]string{
+	"enterprise": "https://api.itglue.com",
+}
```

Comment:
Is this a thing that can change (like other values than `enterprise`)? If not, you could remove this map and just have the URL as a `const apiURL = "https://api.itglue.com"` and use `apiURL` directly.

Reply:
fixed here https://github.com/refractionPOINT/usp-adapters/pull/107/commits/5bb927e68b9a124a149bc90b472daf851cd80c13